modules/apps/app-builder/app-builder-web/test/js/utils/toast.es.js

```diff
@@ -39,7 +39,7 @@ describe('toast', () => {
 			message: Liferay.Language.get(
 				'your-request-completed-successfully'
 			),
-			title: `${Liferay.Language.get('success')}:`,
+			title: `${Liferay.Language.get('success')}`,
```

Comment:
Unnecessary template literal.

Reply:
I love when you get picky... I usually means we're getting closer 😂

![looks_wrong](https://user-images.githubusercontent.com/905006/89510365-c4043180-d7d0-11ea-98ad-7e3920333c81.gif)

Reply:
Next step is:

> Can you break this into two parts and resend?

Reply:
Don't see why not! :)